fast_api/routes/organization.py

```diff
@@ -342,3 +331,48 @@ def archive_admin_message(
 def set_language_display(request: Request, body: UserLanguageDisplay = Body(...)):
     user_manager.update_user_language_display(body.user_id, body.language_display)
     return pack_json_result({"data": {"changeUserLanguageDisplay": {"ok": True}}})
+
+
+@router.post("/mapped-sorted-paginated-users")
+def get_mapped_sorted_paginated_users(
+    request: Request, body: MappedSortedPaginatedUsers = Body(...)
+):
+    active_users = user_manager.get_active_users(body.filter_minutes, None)
+    active_users = [
+        {
+            "id": str(user.id),
+            "lastInteraction": (
+                user.last_interaction.isoformat() if user.last_interaction else None
+            ),
+            "role": user.role,
+            "organizationName": (
+                organization_manager.get_organization_by_id(str(user.organization_id))[
+                    "name"
+                ]
+                if user.organization_id
+                else ""
+            ),
+        }
+        for user in active_users
+    ]
+    active_users = {user["id"]: user for user in active_users}
+
+    data = user_manager.get_mapped_sorted_paginated_users(
+        active_users, body.sort_key, body.sort_direction, body.offset, body.limit
+    )
+    return pack_json_result(
+        {
+            "data": {
+                "mappedSortedPaginatedUsers": data,
+                "fullCountUsers": len(active_users),
+            }
+        },
+        wrap_for_frontend=False,
+    )
+
+
+@router.delete("/delete-user")
+def delete_user(request: Request, body: DeleteUserBody = Body(...)):
+    auth_manager.check_admin_access(request.state.info)
+    user_manager.delete_user(body.user_id)
+    return pack_json_result({"data": {"deleteUser": {"ok": True}}})
```

Comment:
If this is a new entpoint we do need to mimic the response pattern(date + query name) of graphql because it was never an query nedpoint of it